.github/workflows/main.yml

```diff
@@ -154,7 +154,7 @@ jobs:
       with:
         fetch-depth: 50
     - name: Start PostgreSQL PGV=${{ matrix.pg_version }} TZ=${{ matrix.server_tz }} XA=${{ matrix.xa }} SSL=${{ matrix.ssl }} SCRAM=${{ matrix.scram }} CREATE_REPLICAS=${{ matrix.replication }}
-      working-directory: docker/postgres-server
+      working-directory: ${{ matrix.pg_version == 'HEAD' && 'docker/postgres-head' || 'docker/postgres-server' }}
```

Comment:
There's a lot of random failures building and testing against HEAD so we should ignore errors. We'll see have the details in actions to see what happened, but no point in CI randomly turning red because HEAD itself is broken. 

That's what the omni action would do: https://github.com/pgjdbc/pgjdbc/blob/450488c142fdc368cab54e8257407603acc18c4f/.github/workflows/omni.yml#L374

Also, we should probably disable HEAD entirely for PRs and enable it always for master and any daily runs.